### PR TITLE
*: use a path-style dependency on curl-ca-bundle

### PR DESCRIPTION
**Pull request by catap**
#### Description

Closes: https://trac.macports.org/ticket/69054

###### Type(s)

- [x] bugfix

###### Tested on

macOS 12.7.1 21G920 x86_64
Xcode 14.2 14C18

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Review by cooljeanius — APPROVED** [review]

**Comment by catap** [comment]
@cooljeanius if you have accces, may I ask you to merge it?

**Comment by cooljeanius** [comment]
> @cooljeanius if you have accces, may I ask you to merge it?

No, I don't have access, sorry. If I did, I'd merge it, but I don't.

**Comment by reneeotten** [comment]
as usual we'll wait a little bit to give @l2dy a chance to respond, although the change looks fine indeed

File: net/openntpd/Portfile [hunk 1]
```diff
@@ -51,7 +51,8 @@ post-activate {
 }
 
 variant https description {Enable HTTPS Constraint Functionality} {
-    depends_lib-append      path:lib/libtls.dylib:libressl port:curl-ca-bundle
+    depends_lib-append      path:lib/libtls.dylib:libressl \
+                            path:share/curl/curl-ca-bundle.crt:curl-ca-bundle
```
Comment (l2dy) [inline net/openntpd/Portfile:55]: Does this port need a revbump?

Comment (catap) [reply net/openntpd/Portfile:55]: It isn't default variant and I think that not. But it is your call and if you think that it's nessesary, I can add it. Shall I?

Comment (l2dy) [reply net/openntpd/Portfile:55]: Yes, please do it.

Comment (catap) [reply net/openntpd/Portfile:56]: Here it is

**Comment by l2dy** [comment]
Merged. Thank you!

**Pull request merged**